### PR TITLE
Preflight Checks, Support Bundle and Logo

### PR DESCRIPTION
**Pull request by aj-jester**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Thanks for the suggestions! Some of the precheck requirements are steeper than we need, but I'm happy to copy in some of this.

Also already pushed up a logo.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+deps/replicated
```
Comment (MikaelSmith) [inline .gitignore:1]: What generates this?

**Comment by MikaelSmith** [comment]
Ok, I've applied most of them.

File: .gitignore (hunk 1 above)
Comment (aj-jester) [reply .gitignore:1]: Hey, I have a Makefile that creates this, so I added it to the ignore, thanks! 👍

**Pull request closed without merging**